rep-0147.rst

```diff
@@ -275,22 +277,27 @@ Acceleration Interface
 ,,,,,,,,,,,,,,,,,,,,,,
 
 
-The messge to send a velocity command is ``geometry_msgs/Twist`` [#twist]_.
+The messge to send a velocity command is ``geometry_msgs/Twist`` [#twiststamped]_.
```

Comment:
There is a geometry_msgs/AccelStamped message available, since `Twist` represents velocity.